### PR TITLE
feat(global): disable/customize cache (for one request)

### PR DESCRIPTION
**Pull request by rimiti**
## Purpose

As mentioned into the README, this PR aims to:
- disable cache for one request
- customize cache duration for one request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ If you want to customize them, you can also do:
 axiosRedis.methodsToCache = [EHttpMethod.GET];
 ```
 
-### What's the key structure?
+### Key structure
 
 As default, keys have bellow pattern:
```
Comment (Theo-Le-Donne) [inline README.md]: typo in bellow  => `By default, redis keys follow this pattern`
un détail 😉

File: README.md [hunk 2]
```diff
@@ -72,7 +72,29 @@ Example:
...
+
+```ts
+// This request will be cached during 90000ms
+await axiosInstance.get('/user?ID=12345', { 
+  headers: {
+    'Axios-Redis-Cache-Duration': 90000,
+  },
+});
+```
+
```
Comment (Theo-Le-Donne) [inline README.md]: 👍

File: src/classes/index.ts [hunk 3]
```diff
@@ -42,10 +42,21 @@ export class AxiosRedis {
    * @description Caches data.
    * @param {string} key
    * @param {AxiosResponse} data
+   * @param {undefined | number} durationInMS
    * @returns Promise<string>
    */
-  setCache(key: string, data: AxiosResponse): Promise<string> {
-    return this.redisSetAsync(key, flatted.stringify(data), ERedisFlag.EXPIRATION, this.config.expirationInMS);
+  async setCache(key: string, data: AxiosResponse, durationInMS: undefined | number): Promise<string> {
+    if (durationInMS === undefined) {
```
Comment (Simobain) [inline src/classes/index.ts]: Possibilité de simplifier: 
durationInMs = durationInMS || this.config.expirationInMS

File: src/classes/index.ts [hunk 4]
```diff
@@ -109,6 +121,8 @@ export class AxiosRedis {
    * @returns {AxiosPromise}
    */
   private fetch(config: AxiosRequestConfig): AxiosPromise {
+    delete config.headers[EAxiosCacheHeaders.CacheDuration];
```
Comment (Theo-Le-Donne) [inline src/classes/index.ts:120]: bien vu

File: src/classes/index.ts (hunk 3 above)
Comment (Simobain) [reply src/classes/index.ts]:
```
if (durationInMS === 0) {
    return ''
} else {
  durationInMs = durationInMS || this.config.expirationInMS
 return this.redisSetAsync(key, flatted.stringify(data), ERedisFlag.EXPIRATION, durationInMS);
}
```

Comment (rimiti) [reply src/classes/index.ts]: Absolutely! Thanks for your review.

**Review by Simobain — APPROVED** [review]

**Review by Simobain — APPROVED** [review]
Amazing !

**Pull request merged**